package/MDAnalysis/analysis/rms.py

```diff
@@ -293,49 +313,63 @@ class RMSD(AnalysisBase):
     Parameters
     ----------
-    atomgroup : :class:`~MDAnalysis.core.groups.AtomGroup` or
-                :class:`~MDAnalysis.core.universe.Universe`
-        MDAnalysis `AtomGroup` or `Universe` with an associated trajectory
-        to be RMSD fit.
-    reference : :class:`~MDAnalysis.core.groups.AtomGroup` or
-                :class:`~MDAnalysis.core.universe.Universe`, optional
-        Reference coordinates, if ``None`` current frame of `atomgroup` is
-        used
-    select : str / dict / tuple (optional)
+    atomgroup : AtomGroup or Universe
+        Group of atoms for which the RMSD is calculated. If a trajectory is
+        associated with the atoms then the computation iterates over the
+        trajectory.
+    reference : AtomGroup or Universe (optional)
+        Group of reference atoms; if ``None`` then the current frame of
+        `atomgroup` is used.
+    select : str or dict or tuple (optional)
         The selection to operate on; can be one of:
 
         1. any valid selection string for
            :meth:`~MDAnalysis.core.groups.AtomGroup.select_atoms` that
-           produces identical selections in *mobile* and *reference*; or
+           produces identical selections in `atomgroup` and `reference`; or
+
+        2. a dictionary ``{'mobile': sel1, 'reference': sel2}`` where *sel1*
+           and *sel2* are valid selection strings that are applied to
+           `atomgroup` and `reference` respectively (the
+           :func:`MDAnalysis.analysis.align.fasta2select` function returns such
+           a dictionary based on a ClustalW_ or STAMP_ sequence alignment); or
 
-        2. a dictionary ``{'mobile':sel1, 'reference':sel2}`` (the
-           :func:`MDAnalysis.analysis.align.fasta2select` function returns
-           such a dictionary based on a ClustalW_ or STAMP_ sequence
-           alignment); or
         3. a tuple ``(sel1, sel2)``
 
-        When using 2. or 3. with *sel1* and *sel2* then these selections
-        can also each be a list of selection strings (to generate a
-        AtomGroup with defined atom order as described under
-        :ref:`ordered-selections-label`).
+        When using 2. or 3. with *sel1* and *sel2* then these selection strings
+        are applied to `atomgroup` and `reference` respectively and should
+        generate *groups of equivalent atoms*.  *sel1* and *sel2* can each also
+        be a *list of selection strings* to generate a
+        :class:`~MDAnalysis.core.groups.AtomGroup` with defined atom order as
+        described under :ref:`ordered-selections-label`).
+
     groupselections : list (optional)
-        A list of selections as described for `select` Each selection
+        A list of selections as described for `select`. Each selection
         describes additional RMSDs to be computed *after the structures
-        have be superpositioned* according to `select`. The output contains
+        have been superpositioned* according to `select`. The output contains
```

Comment:
The wonderful thing about the English language is that you can turn *anything* into a verb.

(German is different and Dutch might be, too: anything can be nounified (hehe).)

One of my favorite Calvin and Hobbes strips: http://www.gocomics.com/calvinandhobbes/1993/01/25/ : **Verbing weirds language.**

![Verbing weirds language.](http://assets.amuniversal.com/6f34d860df950131725e005056a9545d)